app/controllers/admin/questions_controller.rb

```diff
@@ -22,45 +21,76 @@ def new
     def create
       @question = @conference.questions.new(params[:question])
       @question.conference_id = @conference.id
+
+      # We need to authorize the @question after a conference_id has been associated with the question,
+      # because authorization in ability.rb is based on existence of conference_id attribute
+      # (and the controller does not authorize through conference)
       authorize! :create, @question
 
-      if @question.question_type_id == QuestionType.find_by(title: 'Yes/No').id
-        @question.answers = [Answer.find_by(title: 'Yes'), Answer.find_by(title: 'No')]
+      if @question.question_type == QuestionType.find_by(title: 'Yes/No')
+        @question.answers = [ Answer.find_or_create_by(title: 'Yes'), Answer.find_or_create_by(title: 'No') ]
       end
 
       respond_to do |format|
-        if @conference.save
-          format.html { redirect_to admin_conference_questions_path, notice: 'Question was successfully created.' }
+        # Do not automatically associate newly created question with the conference. The new question shall be enabled for the conference manually.
+        if @question.save
+          format.html { redirect_to admin_conference_questions_path(@conference.short_title), notice: 'Question was successfully created.' }
         else
           flash[:error] = "Oops, couldn't save Question. #{@question.errors.full_messages.join('. ')}"
-          format.html { redirect_to admin_conference_questions_path }
+          format.html { redirect_to admin_conference_questions_path(@conference.short_title) }
         end
       end
     end
 
     # GET questions/1/edit
-    def edit
-      if @question.global
-        redirect_to(admin_conference_questions_path(conference_id: @conference.short_title), alert: 'Sorry, you cannot edit global questions. Create a new one.')
-      end
-    end
+    def edit; end
 
     # PUT questions/1
     def update
-      if @question.update_attributes(params[:question])
-        redirect_to(admin_conference_questions_path(conference_id: @conference.short_title), notice: "Question '#{@question.title}' for #{@conference.short_title} successfully updated.")
+      @question.assign_attributes(params[:question])
+
+      if @question.question_type == QuestionType.find_by(title: 'Yes/No')
+        @question.answers = [ Answer.find_or_create_by(title: 'Yes'), Answer.find_or_create_by(title: 'No') ]
+      end
+
+      if @question.save
+        if @question.answers.blank?
```

Comment:
Huh? You save it and then delete it? And without telling the user what happened?